Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN printf "I am running on ${BUILDPLATFORM:-linux/amd64}, building for ${TARGET
 RUN apk add --no-cache chromium && \
     find /usr/lib/chromium/locales -type f ! -name 'en-US.*' -delete
 
+RUN apk add --no-cache dumb-init

```

Comment:
Use this instead to reduce Docker image size:
```
RUN apk add --no-cache chromium dumb-init && \
    find /usr/lib/chromium/locales -type f ! -name 'en-US.*' -delete
`````

Reply:
Thanks for the feedback.

Is it possible to explain the difference between the two? Personal curiosity.

I can understand having all the packages installed via a one liner, but I can't see why the docker image size would be smaller since both use the no-cache parameter. Is it because it will reinstall Chromium locale on the second apk?

Reply:
No => https://github.com/NoahCardoza/CloudProxy/pull/33#issuecomment-729598554

Reply:
I wasn't aware of that, thanks for the info